packages/tpetra/core/src/Tpetra_Distributor.cpp

```diff
@@ -154,11 +155,13 @@ namespace Tpetra {
     makeTimers ();
 #endif // TPETRA_DISTRIBUTOR_TIMERS
 
-    if (debug_) {
+    if (verbose || debug_) {
       TEUCHOS_TEST_FOR_EXCEPTION
```

Comment:
@mhoemmen, I made the check for `verbose` or `debug_`.  I imagine it is conceivable that if `debug_` one would want to write to `out_` (though the preferred would be to write to `out_` only if `verbose`)

Reply:
@tjfulle Does `out_` get initialized in either case?  If so, then I'm OK with this.

Reply:
@mhoemmen, according to [Line 148 of Tpetra_Distributor.cpp](https://github.com/tjfulle/Trilinos/blob/1e78c86eb2313a8535594f4c47a6d5ed785535cb/packages/tpetra/core/src/Tpetra_Distributor.cpp#L148), `out_` is set independent of `debug_` or `verbose` - to either the incoming value of `out` or `Teuchos::getFancyOStream (Teuchos::rcpFromRef (std::err))` if `out` is null.